packages/dashboard/src/components/Wizard/PanelWizard.styles.ts

```diff
@@ -0,0 +1,45 @@
+import { IPanelStyles, ITheme } from 'office-ui-fabric-react';
+import { IWizardStyles } from './Wizard.types';
+import { IPanelWizardStyles, IPanelWizardStyleProps } from './PanelWizard.types';
+
+export const defaultPanelStyleSet = (theme: ITheme): Partial<IPanelStyles> => {
+  return {
+    content: {
+      height: '100%',
```

Comment:
Is content in a flex container? If so I think it would be better to use that versus height:100%

Reply:
It's actually just a div.

However looking at this again I think I should hide overflow by default.